tests/test_dataset_segmentations.py

```diff
@@ -117,12 +196,11 @@ def test_segmentation_entire_dataset(self) -> None:
 
     def test_segmentation_not_supported(self) -> None:
         with self.assertRaises(SegmentationNotSupportedException) as context:
```

Comment:
Why is the `as context` kept here, but removed elsewhere?

Reply:
It should be removed as it is no longer used